app-admin/rasdaemon/rasdaemon-0.6.2-r4.ebuild

```diff
@@ -32,23 +32,20 @@ pkg_setup() {
 }
 
 src_configure() {
-	local myconf=(
+	local myeconf=(
```

Comment:
is there a reason you renamed the variable here?

Reply:
No. I am sorry, this was not intended, I've forgot to disable my vim plugin, since I try to write my own ebuilds the same way. Changed back.